Source/Plugins/EditorModules/ObjectInspector/Modules/ObjectInspector.cs

```diff
@@ -28,7 +23,13 @@ public partial class ObjectInspector : DockContent
 		private ObjectSelection.Category displayCat          = ObjectSelection.Category.None;
 
 		private ExpandState              gridExpandState     = new ExpandState();
-
+		private ObjectInspectorState userSettings;
```

Comment:
Local style: Either remove the table-like alignment from all the fields, or make the new field align with the existing ones, as to not introduce inconsistency.